code/game/objects/items.dm

```diff
@@ -656,17 +656,16 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
 
 /obj/item/proc/set_force_string()
-	var/roundForce = round(force)
-	switch(roundForce)
-		if(0 to 3)
+	switch(force)
+		if(0 to 4)
 			force_string = default_strings[1]
-		if(4 to 6)
+		if(4 to 7)
 			force_string = default_strings[2]
-		if(7 to 9)
+		if(7 to 10)
 			force_string = default_strings[3]
-		if(10 to 11)
+		if(10 to 12)
 			force_string = default_strings[4]
-		if(12 to 20) //12 is the force of a toolbox
+		if(12 to 21) //12 is the force of a toolbox

```

Comment:
this one should actually be 11 to 20(and the above one 10 to 11) so that it catches toolboxes properly.